.rubocop.yml

```diff
@@ -5,6 +5,6 @@ inherit_gem:
   rubocop-jekyll: .rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.7
```

Comment:
Perhaps Ruby 3 will be available sometime within the next few months. Until then, I'm commenting out the Rubocop run in the Travis files.